assets/app/scripts/directives/logViewer.js

```diff
@@ -103,6 +150,24 @@ angular.module('openshiftConsole')
                 DataService.createStream($scope.kind, $scope.name, $scope.context, options);
 
               var lastLineNumber = 0;
+
+              var addLine = function(lineNumber, text) {
+                bufferedLines.push({
+                  lineNumber: lineNumber,
+                  text: text
+                });
+
+                if ($scope.autoScroll) {
+                  // Throttle updates to avoid flicker on bursts of messages,
+                  // but only on autoscroll to avoid an artificial delay when
+                  // the log first loads.
+                  updateThrottled();
```

Comment:
The problem is if the log is complete, the first line displays, then there's a pause, then the rest of the lines display.